### ISSUE TITLE
Smoothen opacity on iOS Touchable

### ISSUE THREAD

**Issue by donni106** [smart-village-solutions/smart-village-app-app#65]
The `TouchableOpacity` for iOS can be smoothen with the following prop `activeOpacity={0.8}`:

https://github.com/ikuseiGmbH/smart-village-app-app/blob/8f00c148db1ceef86b278975db472e986611321e/src/components/Touchable.js#L5

We already used this in https://github.com/ikuseiGmbH/smart-village-app-app/pull/62/files#diff-c419682ce198e95c29ad105c84adf7dbR24. This makes the element, which is touched, less transparent and smoothens the effect.

Maybe this can also be used for our `Button`?

**Comment by donni106** [comment]
Maybe https://reactnavigation.org/docs/elements#platformpressable can be used

**Comment by ardasnturk** [comment]
We use the `TouchableOpacity` component in many places in the app, but we use `activeOpacity={0.8}` in only 2 places (`ChipFilter`, `ImagesCarouselItem`). Do we want to make this update for the start, pause button in `ImagesCarousel`? We can add the `activeOpacity` prop instead of changing the whole component.

**Comment by donni106** [comment]
Yes, that's the idea. We need to find out if it makes sense to do it more generally.
